opacclient/tests/src/main/java/de/geeksfactory/opacclient/apis/WebOpacAtTest.java

```diff
@@ -0,0 +1,45 @@
+package de.geeksfactory.opacclient.apis;
+
+import org.json.JSONObject;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.Objects;
+
+import de.geeksfactory.opacclient.objects.Library;
+import de.geeksfactory.opacclient.searchfields.SearchQuery;
+
+public class WebOpacAtTest {
+
+    WebOpacAt getApi() {
+        WebOpacAt api = new WebOpacAt();
+        Library lib = new Library();
+        lib.setData(new JSONObject(Collections.singletonMap("baseurl", "http://www.stbinnsbruck.web-opac.at")));
+        api.init(lib);
+        return api;
+    }
+
+    @Test
+    public void testParseTotalResults() {
+        assert WebOpacAt.parseTotalResults("Treffer pro Seite  •  1 - 10 von 46 Treffern") == 46;
+        assert WebOpacAt.parseTotalResults("Hits per page  •  1 - 10 of 46") == 46;
+        assert WebOpacAt.parseTotalResults("Sayfa başına sonuç  •  46 sonuçtan 1 - 10") == 46;
+    }
+
+    @Test
+    public void testParseCopyReturn() throws Exception {
+        assert Objects.equals(WebOpacAt.parseCopyReturn("1 (voraussichtl. bis 31.07.2015)"), "31.07.2015");
+
+    }
+
+    @Test
+    public void testSearchFields() throws Exception {
+        getApi().getSearchFields();
```

Comment:
Of all the three ways you proposed, I only like the third one better than testing against a specific library. The good thing is, that we do not have to provide all this unit tests at the same time, we can add them slowly during the next years -- event one unit test is better than none.

However, this leaves the scope of this PR. So @simon04, if you remove those two tests as suggested, I'll go ahead and merge.

PS: There is no reason for not using camel case there, but at least it's not a public API. ;)